e2e/specs/cookie.spec.js

```diff
@@ -0,0 +1,17 @@
+const SingIn = require('../pages/SignIn');
+const CookieManager = require('../cookies/CookieManager');
+let username = 'hapsneeze';
+let password = 'test12345';
+describe('pivotal tracker verify user cookie', () => {
+    let dashboard;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'dashboard' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)